diff --git a/node_modules/typeorm/platform/PlatformTools.js b/node_modules/typeorm/platform/PlatformTools.js
index 70c1a41..2ff7855 100644
--- a/node_modules/typeorm/platform/PlatformTools.js
+++ b/node_modules/typeorm/platform/PlatformTools.js
@@ -35,80 +35,11 @@ class PlatformTools {
         try {
             // switch case to explicit require statements for webpack compatibility.
             switch (name) {
-                /**
-                 * spanner
-                 */
-                case "spanner":
-                    return require("@google-cloud/spanner");
-                /**
-                 * mongodb
-                 */
-                case "mongodb":
-                    return require("mongodb");
-                /**
-                 * hana
-                 */
-                case "@sap/hana-client":
-                    return require("@sap/hana-client");
-                case "@sap/hana-client/extension/Stream":
-                    return require("@sap/hana-client/extension/Stream");
-                case "hdb-pool":
-                    return require("hdb-pool");
-                /**
-                 * mysql
-                 */
-                case "mysql":
-                    return require("mysql");
-                case "mysql2":
-                    return require("mysql2");
-                /**
-                 * oracle
-                 */
-                case "oracledb":
-                    return require("oracledb");
-                /**
-                 * postgres
-                 */
-                case "pg":
-                    return require("pg");
-                case "pg-native":
-                    return require("pg-native");
-                case "pg-query-stream":
-                    return require("pg-query-stream");
-                case "typeorm-aurora-data-api-driver":
-                    return require("typeorm-aurora-data-api-driver");
-                /**
-                 * redis
-                 */
-                case "redis":
-                    return require("redis");
-                case "ioredis":
-                    return require("ioredis");
                 /**
                  * better-sqlite3
                  */
                 case "better-sqlite3":
                     return require("better-sqlite3");
-                /**
-                 * sqlite
-                 */
-                case "sqlite3":
-                    return require("sqlite3");
-                /**
-                 * sql.js
-                 */
-                case "sql.js":
-                    return require("sql.js");
-                /**
-                 * sqlserver
-                 */
-                case "mssql":
-                    return require("mssql");
-                /**
-                 * react-native-sqlite
-                 */
-                case "react-native-sqlite-storage":
-                    return require("react-native-sqlite-storage");
             }
         }
         catch (err) {
